web/src/components/text-input/styles.ts

```diff
@@ -0,0 +1,85 @@
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../tokens'
+import { getColorDefault, getColorHover } from '../checkbox/styles'
+
+export default createUseStyles({
+  textField: {
+    fontFamily: tokens.fontFamily,
+    borderRadius: ({ component }) => {
+      return component === 'input'
+        ? tokens.borderRadiusFormfieldInput
+        : tokens.borderRadiusFormfieldTextarea
+    },
+    borderWidth: tokens.borderWidthFormfield,
+    borderStyle: 'solid',
+    color: tokens.colorTextPrimary,
+    resize: 'none',
+    backgroundColor: ({ isDisabled, isReadOnly }) => {
+      if (isDisabled) return tokens.colorBackgroundSecondary
+      if (isReadOnly) return tokens.colorBackgroundFormfieldReadonly
+      return undefined
+    },
+    cursor: ({ isDisabled }) => {
+      if (isDisabled) return 'not-allowed'
+      return undefined
+    },
+    height: ({ component, maxLength }) => {
+      if (component === 'input') return tokens.sizeFormfieldInput
+      if (maxLength) {
+        return '92px'
+      }
+      return '60px'
+    },
+    boxSizing: 'border-box',
+    padding: () => {
+      return `${tokens.paddingFormfieldTextareaInputtextTopbottom} ${tokens.paddingFormfieldInputtextLeftright}`
+    },
+    paddingRight: ({ maxLength, component }) => {
+      if (maxLength && component === 'input')
+        return `calc(${tokens.paddingFormfieldInputtextCounter} + 50px)`
+
+      return null
```

Comment:
Probably should be replaced with default 0 here. Just will look more clear as I think

Reply:
If we return null/undefined, no value will be set for `paddingLeft`/`paddingRight` that's what we want because `padding` is set and would overwrite the values

Reply:
Thanks for explanation, missed that